sonar-project.properties

```diff
@@ -4,3 +4,5 @@ sonar.analysis.mode=publish
 
 sonar.java.binaries=build/classes
 sonar.java.libraries=build/libs
+
+sonar.coverage.exclusions=src/test
```

Comment:
What I tried was to exclude tests themself from the coverage report (in SonarCloud) but it didn't work. :( Any ideas?